src/seedu/addressbook/Main.java

```diff
@@ -127,5 +128,19 @@ private StorageFile initializeStorage(String[] launchArgs) throws InvalidStorage
         return isStorageFileSpecifiedByUser ? new StorageFile(launchArgs[0]) : new StorageFile();
     }
 
-
+    /**
+     * @param result of the executed command from user
+     * if the result indicates to automatically list addressbook, this function
+     * 'creates' and executes a new list result.
+     * otherwise it does nothing
+     */
```

Comment:
Good that you included a header comment for your new method, but let's keep the format of the header comment consistent. For example, you can refer to the header comment for the `initializeStorage` method just above (lines 121 to 125) to see the standard way to write header comments :)